docs/tutorials/comparison.md

```diff
@@ -77,12 +77,13 @@ Download and install [`vs-jet`](https://github.com/Jaded-Encoding-Thaumaturgy/vs
 ```powershell
 python -m pip install vsjet
 ```
+
 ```powershell
 python -m vsjet latest
 ```
 
-!!!
-As of writing this, installing [`vs-preview`](https://github.com/Jaded-Encoding-Thaumaturgy/vs-preview) directly from PyPI or git is broken. You have to install it using [`vs-jet`](https://github.com/Jaded-Encoding-Thaumaturgy/vs-jet) which will install all JET packages.
+!!!warning
+Currently, installing [`vs-preview`](https://github.com/Jaded-Encoding-Thaumaturgy/vs-preview) directly from PyPI or git is broken. Instead, you have to install it using [`vs-jet`](https://github.com/Jaded-Encoding-Thaumaturgy/vs-jet), which will install all necessary JET packages.
```

Comment:
> which will install all necessary JET packages.
It won't install just the necessary stuff required for vs-preview (2 deps), it will install ALL jet packages regardless